mysql-test/t/percona_log_slow_innodb.test

```diff
@@ -1,4 +1,3 @@
-#
```

Comment:
Please remove the whole file not just the first line

Reply:
@laurynas-biveinis done